src/scripts/parseargs.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import sys
+import settings
+from os import path
+import argparse
+
+
+DEFAULT_TEST_COUNT = 100 # The Default value of --test-count option
```

Comment:
Hide these with '_' prefix.
It might be better to have some kind of `default_setting` class, object or function which holds these values.

Reply:
Maybe you can have a `default.conf` text file which contains one simple JSON object. This object can hold these information, and this module will load the `default.conf` file if there is no such config object which is passed to this `parse_args` function as a parameter.